library/index.js

```diff
@@ -0,0 +1,35 @@
+const   { WebClient }           = require('@slack/web-api'),
+        { createEventAdapter }  = require('@slack/events-api'),
+        { createReadStream }    = require('fs');
+
+
+// get access token from slack bot and set you enviroment variable SLACK_TOKEN using export SLACK_TOKEN="slack token" in your terminal
+const token = process.env.SLACK_TOKEN;
+const slackEvents = createEventAdapter(process.env.SLACK_SIGNING_SECRET);
+const web = new WebClient(token);
+
+// This argument can be a channel ID, a DM ID, a MPDM ID, or a group ID.
+const channelId = 'C01AA5U5BGT';
```

Comment:
This variable doesn't seem being used anywhere, if this is used for test purpose, we could consider put it in unit test  or other place.

Reply:
yeah it was just for testing.